### ISSUE TITLE
snapchat ads are back. 😔 

### ISSUE THREAD

**Issue by Cliffmode2000** [AdguardTeam/AdGuardSDNSFilter#699]
### Steps to reproduce

1. Snapchat beta 11.36.0.28
2.
3.
4.

### Actual behavior

<details><summary>Screenshot:</summary>

</details>

### Your environment

|   Description  |     Value    |
| -------------- | ------------ |
| DNS server in use:| (dns.adguard.com)
| How did you setup DNS configuration:| (System)
| Device model:| (samsung s20 fe 5g)
| Operating system and version:| (android 11)

**Comment by Alex-302** [comment]
Not sure if it possible. Checked it recently, but cannot find domain, which is used for ads only.
Maybe this may help
https://support.snapchat.com/en-US/a/advertising-preferences

**Comment by Cliffmode2000** [comment]
It worked as of a few days ago. Never saw an ad for a few years. So something changed. I can't find anyone online speaking about it. So it was possible. Maybe not now but before it was working perfectly.

**Comment by Cliffmode2000** [comment]
I can't seem to isolate the server ads are coming from. But I'm not that educated with the servers either.

**Comment by Cliffmode2000** [comment]
Seems they maybe gone?

**Comment by Alex-302** [comment]
They just do not show ads everylime. I alreary tried to block everything, and unblock required minimum - ads in all cases.
